### ISSUE TITLE
Editor UI modifications

### ISSUE THREAD

**Issue by jdragojevic** [pculture/unisubs#1492]
Before we start cluttering up the editor with additional feature requests, we need to take a look at the UI / Layout itelf and make some improvement in how we are organizing it.  

Need to:
 - find better home for autopause toggle
 - fit in TED request for edit title prompt in review and transcribe 
 - handle zooming in without wrecking layout
 - ...

**Comment by bendk** [comment]
No comments in a while and there's a new effort underway to improve the editor UI.  I'm going to close this one for now.
